operator/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/kyma-project/manifest-operator/operator
 go 1.18
 
 require (
+	github.com/cyphar/filepath-securejoin v0.2.3
```

Comment:
Not sure why this is there, never seen this before

Reply:
It is referenced here:
https://github.com/adityabhatia/manifest-operator/blob/addDeclaritiveBoilerPlateCode/operator/pkg/util/util.go#L147

Reply:
Out of curiosity, which case does this cover that `filepath.Join` does not in our use case? Also, looking at the package there is lots of utility code which is unnecessary imo and also untested, probably not great to keep it there.